src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -22,16 +22,17 @@
 import hudson.Launcher;
 import hudson.model.Run;
 import hudson.model.TaskListener;
+import hudson.Util;
 
-public class RunMatlabTestsStep extends Step {
+public class RunMatlabTestsStep extends Step implements MatlabBuild{
```

Comment:
I would still suggest just for one common method we need not implement MatlabBuild here see if you could scope it to step execution class or within step class itself

Reply:
Refactored to have duplicate methods in RunMatlabTestBuilder and RunMatlabTestStep